client/blocks/nps-survey/docs/example.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import NpsSurvey from '../';
+import { isEnabled } from 'config';
+
+class NpsSurveyExample extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			wasDismissed: false,
+			wasSubmitted: false,
+			surveyName: null,
+			recommendationValue: null,
+		};
+		this.handleDismissed = this.handleDismissed.bind( this );
+	}
+
+	handleDismissed( event ) {
+		this.setState( {
+			wasDismissed: true,
+			wasSubmitted: event.wasSubmitted,
+			surveyName: event.surveyName,
+			recommendationValue: event.recommendationValue
+		} );
+	}
+
+	render() {
+		// Since this component is still in-progress and unstyled, we will hide it from
```

Comment:
I'd do the `isEnabled` check in `client/devdocs/design/blocks.jsx` instead of showing this message in production.